plonk/src/proof_system/structs.rs

```diff
@@ -82,6 +82,19 @@ pub struct Proof<E: Pairing> {
     pub plookup_proof: Option<PlookupProof<E>>,
 }
 
+/// A proof-linking hint generated by the prover in the course of proving an
+/// R1CS relation
+///
+/// The linking hint contains information about the prover's witness needed to
+/// link the proof to another proof of a different circuit
+#[derive(Debug, Clone)]
+pub struct LinkingHint<E: Pairing> {
+    /// The wire polynomial that encodes the proof-linking gates for the circuit
+    pub linking_wire_poly: DensePolynomial<E::ScalarField>,
+    /// The commitment to the linking poly generated while proving
```

Comment:
```suggestion
    /// The commitment to the wire poly generated while proving
```
this makes it clearer that the commitment is to the wiring polynomial used to encode the linking gates, as opposed to the "linking polynomial" $F(x)$ in the linking PIOP